modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -361,12 +361,58 @@ m|+++10m+++
 |Description
 a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
 Typically, all members of the cluster, including the current server, should be specified in this list.
+The setting configures the endpoints for Discovery service V2.
 |Valid values
 a|A comma-separated list where each element is a socket address in the format of `hostname:port`, `hostname`, or `:port`.
 |Default value
 m|
 |===
 
+[role=label--enterprise-edition, label--new-5.22]
+[[config_dbms.cluster.discovery.v2.endpoints]]
+=== `dbms.cluster.discovery.v2.endpoints`
+
+// label:enterprise-edition[Enterprise Edition]
+
+.dbms.cluster.discovery.v2.endpoints
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
+Typically, all members of the cluster, including the current server, should be specified in this list.
+The setting configures the endpoints for Discovery service V2.
+|Valid values
+a|A comma-separated list where each element is a socket address in the format of `hostname:port`, `hostname`, or `:port`.
+|Default value
+m|
+|===
+
+[role=label--enterprise-edition, label--new-5.22]
+[[config_dbms.cluster.discovery.version]]
+=== `dbms.cluster.discovery.version`
+
+// label:enterprise-edition[Enterprise Edition]
+
+.dbms.cluster.discovery.version
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|This setting allows you to select which discovery service should be started.
```

Comment:
same here

Reply:
This [PR](https://github.com/neo-technology/neo4j/pull/26075) changes the description in the main codebase to reflect these changes.

Reply:
This [PR](https://github.com/neo-technology/neo4j/pull/26075) changes the description in the main codebase to reflect these changes.